src/components/application_manager/src/commands/mobile/add_sub_menu_request.cc

```diff
@@ -60,7 +60,7 @@ void AddSubMenuRequest::Run() {
   const int32_t menu_id =
       (*message_)[strings::msg_params][strings::menu_id].asInt();
   if (app->FindSubMenu(menu_id)) {
-    SDL_ERROR("Menu with id " << menu_id << " is not found.");
+    SDL_ERROR("Menu with id " << menu_id << " is already exist.");
```

Comment:
@LevchenkoS I think that "already exists." or "is already existing." sounds better.